opennms-container/minion/minion-config-schema.yml

```diff
@@ -120,6 +120,64 @@ categories:
             description: "TODO"
             key: "/scv/provider"
             example: "TODO"
+
+  # Java Agent
+  - name: "java"
+    label: "Java"
+    description: "Java configuration parameters."
+    categories:
+      # Java :: Agent
+      - name: "agent"
+        label: "Agent"
+        description: "Configuration parameters for Java agents."
+        categories:
+          # Prometheus JMX exporter
+          - name: "prom-jmx-exporter"
+            label: "Prometheus JMX Exporter"
+            description: "Configuration for the Prometheus JMX Exporter agent."
+            config:
+              - name: "jmxUrl"
+                type:
+                  name: "string"
+                label: "JMX URL"
+                description: "JMX URL connecting to the local JVM."
+                key: "/java/agent/prom-jmx-exporter/jmxUrl"
+                example: "service:jmx:rmi:///jndi/rmi://127.0.0.1:1299/karaf-minion"
+              - name: "username"
+                type:
+                  name: "string"
+                label: "Username"
+                description: "Connection username to establish the JMX connection."
+                key: "/java/agent/prom-jmx-exporter/username"
+                example: "admin"
+              - name: "password"
+                type:
+                  name: "string"
+                label: "Password"
+                description: "Connection password to establish the JMX connection."
+                key: "/java/agent/prom-jmx-exporter/password"
+                example: "admin"
+              - name: "lowerCaseOutputName"
+                type:
+                  name: "boolean"
+                label: "Lowercase output name"
+                description: "Lowercase the output metric name."
+                key: "/java/agent/prom-jmx-exporter/lowerCaseOutputName"
+                example: "true"
+              - name: "lowercaseOutputLabelNames"
+                type:
+                  name: "boolean"
+                label: "Lowercase output label names"
+                description: "Lowercase the output metric label names."
+                key: "/java/agent/prom-jmx-exporter/lowercaseOutputLabelNames"
+                example: "true"
+              - name: "whitelistObjectNames"
+                type:
+                  name: "string"
```

Comment:
This is what I'm proposing changing to `set` (and then adding the strategy and element information).